platform/ios/src/MGLMapView.mm

```diff
@@ -3550,8 +3550,12 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
         {
             id <MGLAnnotation> annotation = [self annotationWithTag:annotationTag];
             NSAssert(annotation, @"Unknown annotation found nearby tap");
```

Comment:
Should we consider that a bug, or is it an edge case we want to fail gracefully around from now on?